### ISSUE TITLE
how to use host for files editing?

### ISSUE THREAD

**Issue by sivajik34** [GordonLesti/docker-magento1#1]
I'm using windows as a host.I want to use IDE in windows.
when i run following command ,its deleting all files in /opt/magento directory in ubuntu machine.
docker run -d -v \c\Users\Admin\docker-magento\magento:/opt/magento -p 80:80 --name mtfcont 8196e89a13f0.
Can you help me on this?

**Comment by GordonLesti** [comment]
The directory `/opt/magento` is the installation directory of Magento in the Debian container. I guess your volume mount to `/opt/magento` is deleting Magento.

**Comment by sivajik34** [comment]
yes i understood.Is there any way to edit magento source code  from host machine?

**Comment by GordonLesti** [comment]
no, unfortunately not. Maybe I will change that in future.
